worldgenedit.lua

```diff
@@ -0,0 +1,112 @@
+--Edit certain worldgen parameters while it is running.
+local gui = require 'gui'
+local widgets = require 'gui.widgets'
+local wv = df.global.gview.view.child.child
```

Comment:
I think this was the second widgets script I worked on ever so I didn't actually know those tricks, but yeah I'll pop those in when I get some food in me.